debugger/loadprinter.ml

```diff
@@ -89,15 +88,17 @@ let loadfile ppf name =
 (* Note: evaluation proceeds in the debugger memory space, not in
    the debuggee. *)
 
-let rec eval_path = function
-    Pident id -> Symtable.get_global_value id
-  | Pdot(p, _, pos) -> Obj.field (eval_path p) pos
-  | Papply _ -> fatal_error "Loadprinter.eval_path"
+let rec eval_address = function
+  | Env.Aident id -> Symtable.get_global_value id
+  | Env.Adot(addr, pos) -> Obj.field (eval_address addr) pos
 
 (* PR#7258: get rid of module aliases before evaluating paths *)
 
-let eval_path path =
-  eval_path (Env.normalize_path (Some Location.none) Env.empty path)
+let eval_value_path env path =
```

Comment:
While rebasing #2131 on top of this one, I found this spot.  Does the comment above about PR#7258 still applies?  Does the new code handle module aliases properly?   (There is a similar stuff in toploop.ml and opttoploop.ml.)

Reply:
I should have removed that comment. The `find_foo_address` functions already handle module aliases so there is no need to normalize paths before calling them.

Reply:
In a case like that, you can push a direct commit to remove the comment, no need for a new PR.

Reply:
Done